### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by EZzer0** [ScoopInstaller/Scoop#5633] [Help and Questions]
Remove-Item : The path "D:APP\Scoop\apps\scoop\new" could not be found because the path does not exist.
Location D:APP\Scoop\apps\scoop\current\libexec\scoop-update.ps1:79 characters: 13
            Remove-Item $newdir -Force -Recurse
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:APP\Scoop\apps\scoop\new:String) [Remove-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand


This error is preventing me from updating

**Comment by rashil2000** [comment]
Can you try creating that empty folder (new) once?
